schemas/context/advertising-timed-asset-reference.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/advertising-timed-asset-reference",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Ad Asset Reference",
+  "type": "object",
+  "description": "Asset information about the ad captured by the experience event.",
+  "meta:extensible": true,
+  "definitions": {
+    "adAssetReference": {
+      "properties": {
+        "@id": {
+          "title": "Ad ID",
+          "type": "string",
+          "description": "Identifier of the ad asset."
```

Comment:
The Ad ID is provided by Media Analytics customers when they track ads. It might be an internal ID from their CMS system, it might be from their ad provider (e.g. Freewheel) - there is no standardisation around this.

Reply:
OK, then please clarify that in the description.

Reply:
Improved - Closed